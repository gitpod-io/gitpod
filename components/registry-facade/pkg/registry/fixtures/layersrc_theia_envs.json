{
    "sourceRef": "eu.gcr.io/gitpod-core-dev/build/theia-ide:cw-harmonise-regfac-env.1",
    "content": {
        "6ca02aaaf7cb97a1a5c62f91eb1302b05ee79cd65459477c6ae1c13faa14f963": "eyJjcmVhdGVkIjoiMjAyMS0wMi0wMlQxMDozOToxMy44MDkwMDgwMTFaIiwiYXJjaGl0ZWN0dXJlIjoiYW1kNjQiLCJvcyI6ImxpbnV4IiwiY29uZmlnIjp7IkVudiI6WyJQQVRIPS91c3IvbG9jYWwvc2JpbjovdXNyL2xvY2FsL2JpbjovdXNyL3NiaW46L3Vzci9iaW46L3NiaW46L2JpbiIsIkdJVFBPRF9CVUlMVF9JTl9QTFVHSU5TPS90aGVpYS9ub2RlX21vZHVsZXMvQGdpdHBvZC9naXRwb2QtaWRlL3BsdWdpbnMvIiwiR0lUUE9EX0VOVl9BUFBFTkRfUEFUSD0vaWRlL2JpbjoiLCJHSVRQT0RfRU5WX1NFVF9FRElUT1I9Z3Agb3BlbiAtdyIsIkdJVFBPRF9FTlZfU0VUX1ZJU1VBTD1ncCBvcGVuIC13IiwiR0lUUE9EX0VOVl9TRVRfR0lUX0VESVRPUj1ncCBvcGVuIC13Il0sIldvcmtpbmdEaXIiOiIvaWRlLyJ9LCJyb290ZnMiOnsidHlwZSI6ImxheWVycyIsImRpZmZfaWRzIjpbInNoYTI1NjowMDY5YTFhN2IxNjBkYTliYzA0N2JiZTU5YjQ5ZjE1NmNmYWEzMGE5ZWRkMjg1MjExMTJmNjEzZDc4MDAxMTU2Iiwic2hhMjU2OmY1ODVhODE4ODEzMzQ5YWFlNWIwMTE1YTI0MjAyZjFjMmEwNTA5MWFjMzc5OGYzNDkwYWYzYTkxZGExYmVjZGQiLCJzaGEyNTY6NWI0YTI1ZDE3NjJlNWE4NTI4M2JkMmRiZTdjZTc0OWE1Yjg4YjM4NWNmN2QwOWY3NTAyN2M2ODc1NTI5YzZlNyIsInNoYTI1NjowMjM5ZDYyMWM1YWUwMTE5YjQ1Mzg0OTY4MWZjZmUzM2I0OGEzNzQ4ZmM4ZmY3OTE0NWYyNjAzOTFiMjdlZTMzIiwic2hhMjU2OmFhZjYzYTg4NjU3MWQxYWVmOTMxN2ViNWE0MDJiYzBhNDYyNGI1N2UzNTdhY2Y3YmFmNGZhMDRhMzllNzc4ZjEiLCJzaGEyNTY6YzFkZDIzMWNlNDU4MzcwZWUyYTkzNDBlYTU5ZDlkODdlOTE2YTUxY2U3ODZhNjEwMDNjZmQ3ODdmNWExYWVlZSIsInNoYTI1NjpkNDlhYmIyYzIxMmYyMWZjNTcwM2MxMGIxMzMzOGYzMzU1Mjc5ZTkzYjQ5MDFmMWJkMDIyNTI2NmU4N2JjNmZiIiwic2hhMjU2OjllNmMzZDEzY2RhMzQ1NGRmNDA4ODRmYzhiMTc2NjNhMGU0YTA2Y2E0Mjk3NTRhNTBkMjQ4YmFlOTk0MGY4ZmYiXX0sImhpc3RvcnkiOlt7ImNyZWF0ZWQiOiIyMDIxLTAyLTAyVDEwOjM5OjA3LjY0OTE3NjY3M1oiLCJjcmVhdGVkX2J5IjoiL2Jpbi9zaCAtYyAjKG5vcCkgQ09QWSBkaXI6NDE5MjU4YmY0ZjJlNDg2N2I2ZjIxOGUxZTIyNjY3MWQzNGY2OGVjYmRlN2JmZDBlNzBhMmEzYThkMGU0ZThkOSBpbiAvdGhlaWEvICJ9LHsiY3JlYXRlZCI6IjIwMjEtMDItMDJUMTA6Mzk6MDkuNzU2MDcxODU3WiIsImNyZWF0ZWRfYnkiOiIvYmluL3NoIC1jICMobm9wKSBXT1JLRElSIC9pZGUvIn0seyJjcmVhdGVkIjoiMjAyMS0wMi0wMlQxMDozOTowOS45NzIzMDUyNjRaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApIENPUFkgZmlsZToyYTVmNWFkZGE2YmQxMzJkY2ZhZTRhZDYwZGE2ZjlmMTZmOGYzNzY1YzRmYzU1YjZiOTQ2MGM4YjI3OTk4ZWRlIGluIC9pZGUvICJ9LHsiY3JlYXRlZCI6IjIwMjEtMDItMDJUMTA6Mzk6MTAuMjkxOTc3OTlaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApICBFTlYgR0lUUE9EX0JVSUxUX0lOX1BMVUdJTlM9L3RoZWlhL25vZGVfbW9kdWxlcy9AZ2l0cG9kL2dpdHBvZC1pZGUvcGx1Z2lucy8iLCJlbXB0eV9sYXllciI6dHJ1ZX0seyJjcmVhdGVkIjoiMjAyMS0wMi0wMlQxMDozOToxMS4yMjgzMTMzNjlaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApIENPUFkgZGlyOjUxN2NhY2RjNzNhZmVkZWIxNDkyNjBkNzI3NjkzOTI0NmRmZmE4YWFmN2NkZTQ5YTNiYTBlMjE3NGI0NDdlYjMgaW4gL3RoZWlhL25vZGVfbW9kdWxlcy9AZ2l0cG9kL2dpdHBvZC1pZGUvcGx1Z2lucy8gIn0seyJjcmVhdGVkIjoiMjAyMS0wMi0wMlQxMDozOToxMi4xMjQ5NTA4NzdaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApIENPUFkgZmlsZTo4MTFmMTU1OTgwM2I4YWQ1NTFhNThjMDBhNDNmYjdmNWQyNmNjMTE2OWU4NTVhZTUyYzg2ODk1OTgzMDU3M2ZjIGluIC90aGVpYS9zdXBlcnZpc29yICJ9LHsiY3JlYXRlZCI6IjIwMjEtMDItMDJUMTA6Mzk6MTIuNDQzNzkwODEyWiIsImNyZWF0ZWRfYnkiOiIvYmluL3NoIC1jICMobm9wKSBDT1BZIG11bHRpOjYzOGI4NzRiNGNmZTVlNzhjNTljNDc2OTdmYWQ5YjFjYjk2YzhhODhhMzM0OGNiMzM3NGZjYWFhZWQ4MTUxMjcgaW4gL3RoZWlhLyAifSx7ImNyZWF0ZWQiOiIyMDIxLTAyLTAyVDEwOjM5OjEyLjY1NzkxNTQ1NloiLCJjcmVhdGVkX2J5IjoiL2Jpbi9zaCAtYyAjKG5vcCkgQ09QWSBtdWx0aTo0ZDAzNzBmMWEzMjg2MTE0YTY5MWM2YzIzOGNiOTA2YjRmMDUyZGM2ZTczMDhlMzI1YjA5YTVmYzIzNDk2ZTkyIGluIC90aGVpYS8gIn0seyJjcmVhdGVkIjoiMjAyMS0wMi0wMlQxMDozOToxMi44MTI4OTQzNTJaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApIENPUFkgZGlyOmU3NGM3YzdlZTE0M2I0YmMyYzVmYmIwYzlkZTRkZmI5MTNhNGJjODQ3ZTdhZmQ5OTNjMGMyOGRiMjM0MWViMTIgaW4gL2lkZS9iaW4gIn0seyJjcmVhdGVkIjoiMjAyMS0wMi0wMlQxMDozOToxMy4wNzg4OTA5MTdaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApICBFTlYgR0lUUE9EX0VOVl9BUFBFTkRfUEFUSD0vaWRlL2JpbjoiLCJlbXB0eV9sYXllciI6dHJ1ZX0seyJjcmVhdGVkIjoiMjAyMS0wMi0wMlQxMDozOToxMy4zMjE4NzQ4MjNaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApICBFTlYgR0lUUE9EX0VOVl9TRVRfRURJVE9SPWdwIG9wZW4gLXciLCJlbXB0eV9sYXllciI6dHJ1ZX0seyJjcmVhdGVkIjoiMjAyMS0wMi0wMlQxMDozOToxMy41NjU0OTc5NTVaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApICBFTlYgR0lUUE9EX0VOVl9TRVRfVklTVUFMPWdwIG9wZW4gLXciLCJlbXB0eV9sYXllciI6dHJ1ZX0seyJjcmVhdGVkIjoiMjAyMS0wMi0wMlQxMDozOToxMy44MDkwMDgwMTFaIiwiY3JlYXRlZF9ieSI6Ii9iaW4vc2ggLWMgIyhub3ApICBFTlYgR0lUUE9EX0VOVl9TRVRfR0lUX0VESVRPUj1ncCBvcGVuIC13IiwiZW1wdHlfbGF5ZXIiOnRydWV9XX0=",
        "ee83266d1df23dae9035a744b45477a9335215ce923666df509704e4b75b8883": "eyJzY2hlbWFWZXJzaW9uIjoyLCJjb25maWciOnsibWVkaWFUeXBlIjoiYXBwbGljYXRpb24vdm5kLmRvY2tlci5jb250YWluZXIuaW1hZ2UudjEranNvbiIsImRpZ2VzdCI6InNoYTI1Njo2Y2EwMmFhYWY3Y2I5N2ExYTVjNjJmOTFlYjEzMDJiMDVlZTc5Y2Q2NTQ1OTQ3N2M2YWUxYzEzZmFhMTRmOTYzIiwic2l6ZSI6NDE5MX0sImxheWVycyI6W3sibWVkaWFUeXBlIjoiYXBwbGljYXRpb24vdm5kLmRvY2tlci5pbWFnZS5yb290ZnMuZGlmZi50YXIuZ3ppcCIsImRpZ2VzdCI6InNoYTI1Njo4MzEwYWM3MDA2NWRhZTZjNGUzYzgwMjlmMjI3ZDQ3ZWRlOTc4M2U2ZDRiNzM4YTRlOTIwMmUzYTM2ZjRlNTljIiwic2l6ZSI6ODYyNTgzNzh9LHsibWVkaWFUeXBlIjoiYXBwbGljYXRpb24vdm5kLmRvY2tlci5pbWFnZS5yb290ZnMuZGlmZi50YXIuZ3ppcCIsImRpZ2VzdCI6InNoYTI1NjpmYTFjNjc3ZWY1OGQwOTE2OGY1MzQ0MmY4NmJhODQwZGRhMWUwMTZkODA4NTZiYzkxZTI0Yjk5ODM2NzExYTBmIiwic2l6ZSI6MTI2fSx7Im1lZGlhVHlwZSI6ImFwcGxpY2F0aW9uL3ZuZC5kb2NrZXIuaW1hZ2Uucm9vdGZzLmRpZmYudGFyLmd6aXAiLCJkaWdlc3QiOiJzaGEyNTY6MTE2MjU0OTNlMjBkZTgwYzdkYmQ4NzIwOTdjYWI1YmM2ZjBiNzhmYWYyM2Y0NGM5ZDZmOGY1YmE3M2Q0ZmViYyIsInNpemUiOjI1N30seyJtZWRpYVR5cGUiOiJhcHBsaWNhdGlvbi92bmQuZG9ja2VyLmltYWdlLnJvb3Rmcy5kaWZmLnRhci5nemlwIiwiZGlnZXN0Ijoic2hhMjU2OmFiYWJhZGUwNzhhNmIwM2E5ZjhmNmE3NjU4YWY4ZDU4ZGFlNDUxZTQwZmJkZWI1MGI4NTVlN2RhNjc5OThhM2YiLCJzaXplIjoxMjgwNTcyODl9LHsibWVkaWFUeXBlIjoiYXBwbGljYXRpb24vdm5kLmRvY2tlci5pbWFnZS5yb290ZnMuZGlmZi50YXIuZ3ppcCIsImRpZ2VzdCI6InNoYTI1NjpmOWMwYjlkNDk1MGQ3YjcxNmUwYzczODAwMmQwYTRjNmQwNmM2NjBmM2QxNzg3NjM5YzU2NDA2YTYzMjRkODRkIiwic2l6ZSI6NTk1ODkxMX0seyJtZWRpYVR5cGUiOiJhcHBsaWNhdGlvbi92bmQuZG9ja2VyLmltYWdlLnJvb3Rmcy5kaWZmLnRhci5nemlwIiwiZGlnZXN0Ijoic2hhMjU2OjQ1OTM2NDJkN2E1MzgwNWY4OGZkYzE1NTA2ZjI1NTA2MjkyNjE3MGFiMDkzODRmYjM3NGM4NzRjZjdiYzgxYjgiLCJzaXplIjo1NDAxMDU1fSx7Im1lZGlhVHlwZSI6ImFwcGxpY2F0aW9uL3ZuZC5kb2NrZXIuaW1hZ2Uucm9vdGZzLmRpZmYudGFyLmd6aXAiLCJkaWdlc3QiOiJzaGEyNTY6ZjBjOWRkMTI1MDYzODkxMDBkNzA5YjI2NGY3YmM4OTIyZjRmZGFiYmU2MmY0M2JlMzEyMTU3YmExMzdiMjI0YSIsInNpemUiOjMzM30seyJtZWRpYVR5cGUiOiJhcHBsaWNhdGlvbi92bmQuZG9ja2VyLmltYWdlLnJvb3Rmcy5kaWZmLnRhci5nemlwIiwiZGlnZXN0Ijoic2hhMjU2OjQzZDM4ODIyNGI2MWNlNDgyNDhiZDk0MzIwNDIwMWQ4MDkyZmE1YmFlYWY2MGM1MDVlMGY3M2NjMmY1ODQyZjAiLCJzaXplIjoyMTh9XX0=",
        "eu.gcr.io/gitpod-core-dev/build/theia-ide:cw-harmonise-regfac-env.1": "eyJtZWRpYVR5cGUiOiJhcHBsaWNhdGlvbi92bmQuZG9ja2VyLmRpc3RyaWJ1dGlvbi5tYW5pZmVzdC52Mitqc29uIiwiZGlnZXN0Ijoic2hhMjU2OmVlODMyNjZkMWRmMjNkYWU5MDM1YTc0NGI0NTQ3N2E5MzM1MjE1Y2U5MjM2NjZkZjUwOTcwNGU0Yjc1Yjg4ODMiLCJzaXplIjoxOTkyfQ=="
    },
    "envs": null
}