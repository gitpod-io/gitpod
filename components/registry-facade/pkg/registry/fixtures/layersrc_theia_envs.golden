{
    "error": "",
    "layer": [
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:8310ac70065dae6c4e3c8029f227d47ede9783e6d4b738a4e9202e3a36f4e59c",
                "size": 86258378
            },
            "DiffID": "sha256:0069a1a7b160da9bc047bbe59b49f156cfaa30a9edd28521112f613d78001156"
        },
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:fa1c677ef58d09168f53442f86ba840dda1e016d80856bc91e24b99836711a0f",
                "size": 126
            },
            "DiffID": "sha256:f585a818813349aae5b0115a24202f1c2a05091ac3798f3490af3a91da1becdd"
        },
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:11625493e20de80c7dbd872097cab5bc6f0b78faf23f44c9d6f8f5ba73d4febc",
                "size": 257
            },
            "DiffID": "sha256:5b4a25d1762e5a85283bd2dbe7ce749a5b88b385cf7d09f75027c6875529c6e7"
        },
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:ababade078a6b03a9f8f6a7658af8d58dae451e40fbdeb50b855e7da67998a3f",
                "size": 128057289
            },
            "DiffID": "sha256:0239d621c5ae0119b453849681fcfe33b48a3748fc8ff79145f260391b27ee33"
        },
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:f9c0b9d4950d7b716e0c738002d0a4c6d06c660f3d1787639c56406a6324d84d",
                "size": 5958911
            },
            "DiffID": "sha256:aaf63a886571d1aef9317eb5a402bc0a4624b57e357acf7baf4fa04a39e778f1"
        },
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:4593642d7a53805f88fdc15506f255062926170ab09384fb374c874cf7bc81b8",
                "size": 5401055
            },
            "DiffID": "sha256:c1dd231ce458370ee2a9340ea59d9d87e916a51ce786a61003cfd787f5a1aeee"
        },
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:f0c9dd12506389100d709b264f7bc8922f4fdabbe62f43be312157ba137b224a",
                "size": 333
            },
            "DiffID": "sha256:d49abb2c212f21fc5703c10b13338f3355279e93b4901f1bd0225266e87bc6fb"
        },
        {
            "Descriptor": {
                "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
                "digest": "sha256:43d388224b61ce48248bd943204201d8092fa5baeaf60c505e0f73cc2f5842f0",
                "size": 218
            },
            "DiffID": "sha256:9e6c3d13cda3454df40884fc8b17663a0e4a06ca429754a50d248bae9940f8ff"
        }
    ],
    "envs": [
        "PATH=/ide/bin:",
        "EDITOR=gp open -w",
        "VISUAL=gp open -w",
        "GIT_EDITOR=gp open -w"
    ]
}